.azure/gpu-tests-pytorch.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         set -e
         pip install -r requirements/pytorch/strategies.txt \
                     -r requirements/_integrations/strategies.txt \
-          --find-links ${TORCH_URL}
+          --find-links ${TORCH_URL} -U --upgrade-strategy eager
```

Comment:
As said in the past, we cannot use the eager strategy. This will update our dependencies' dependencies and create broken or unexpected environments.

For example, the PT strategies job is accidentally upgrading torch 1.12 to 1.13 because of this

Reply:
well, it will be the same env as a user installing from the ground, so we shall be aware of it...

Reply:
It's not the same because the user generally does `pip install lightning` which includes all our dependencies together in a single install call.

But here we are eager upgrading only the strategy installations. Eager could be used if in CI we did a single `pip` install command. But that's not the case here.

If the user installed lightning in steps, and used eager mode in some of them, he could also break the dependency resolution. This is what's happening here.